### PR TITLE
Add update

### PR DESCRIPTION
**Pull request by superboyiii**
Reference: https://github.com/neo-project/proposals/pull/136

### CODE REVIEW

File: neo-cli/CLI/MainService.Contracts.cs [hunk 1]
```diff
@@ -35,11 +36,82 @@ private void OnDeployCommand(string filePath, string manifestPath = null)
...
+
+            Transaction tx = new Transaction
+            {
+                Signers = signers,
+                Attributes = Array.Empty<TransactionAttribute>(),
+                Witnesses = Array.Empty<Witness>(),
+            };
+
+            try
+            {
```
Comment (shargon) [inline neo-cli/CLI/MainService.Contracts.cs:86]: Include LoadUpdateScript?

File: neo-cli/CLI/MainService.Contracts.cs [hunk 2]
```diff
@@ -35,11 +36,82 @@ private void OnDeployCommand(string filePath, string manifestPath = null)
...
             SignAndSendTx(NeoSystem.StoreView, tx);
         }
 
+        /// <summary>
+        /// Process "update" command
+        /// </summary>
+        /// <param name="filePath">File path</param>
+        /// <param name="manifestPath">Manifest path</param>
+        [ConsoleCommand("update", Category = "Contract Commands")]
+        private void OnUpdateCommand(UInt160 scriptHash, string filePath, string manifestPath = null, UInt160 sender = null, UInt160[] signerAccounts = null)
```
Comment (nicolegys) [inline neo-cli/CLI/MainService.Contracts.cs:55]: Where to pass `data`?
https://github.com/neo-project/proposals/pull/136/files#diff-249a6779702f4ea632b82d61a2d7879084dc130f6e1ed560f066bb5f36bba3b9R71-R74

Comment (nicolegys) [inline neo-cli/CLI/MainService.Contracts.cs:55]: I think `sender` should be a required parameter

File: neo-cli/CLI/MainService.Contracts.cs (hunk 1 above)
Comment (superboyiii) [reply neo-cli/CLI/MainService.Contracts.cs:85]: Done.

File: neo-cli/CLI/MainService.Contracts.cs (hunk 2 above)
Comment (superboyiii) [reply neo-cli/CLI/MainService.Contracts.cs]: > Where to pass `data`?
> https://github.com/neo-project/proposals/pull/136/files#diff-249a6779702f4ea632b82d61a2d7879084dc130f6e1ed560f066bb5f36bba3b9R71-R74

RegisterCommandHander can't handle `object`, seems SDK is better to do this if want to add data.

Comment (superboyiii) [reply neo-cli/CLI/MainService.Contracts.cs]: > I think `sender` should be a required parameter

Done

**Review by nicolegys — APPROVED** [review]
test pass

**Comment by superboyiii** [comment]
Please review again. @shargon

File: neo-cli/CLI/MainService.Contracts.cs [hunk 3]
```diff
@@ -35,11 +36,82 @@ private void OnDeployCommand(string filePath, string manifestPath = null)
             UInt160 hash = SmartContract.Helper.GetContractHash(tx.Sender, nef.CheckSum, manifest.Name);
 
             Console.WriteLine($"Contract hash: {hash}");
-            Console.WriteLine($"Gas: {new BigDecimal((BigInteger)tx.SystemFee, NativeContract.GAS.Decimals)}");
-            Console.WriteLine();
+            Console.WriteLine($"Gas consumed: {new BigDecimal((BigInteger)tx.SystemFee, NativeContract.GAS.Decimals)}");
+            Console.WriteLine($"NetFee: {new BigDecimal((BigInteger)tx.NetworkFee, NativeContract.GAS.Decimals)}");
+            Console.WriteLine($"You will pay: {new BigDecimal((BigInteger)(tx.SystemFee + tx.NetworkFee), NativeContract.GAS.Decimals)} GAS");
```
Comment (shargon) [inline neo-cli/CLI/MainService.Contracts.cs:41]:
```suggestion
            Console.WriteLine($"Network fee: {new BigDecimal((BigInteger)tx.NetworkFee, NativeContract.GAS.Decimals)}");
            Console.WriteLine($"Total fee: {new BigDecimal((BigInteger)(tx.SystemFee + tx.NetworkFee), NativeContract.GAS.Decimals)} GAS");
```
?

Comment (shargon) [reply neo-cli/CLI/MainService.Contracts.cs]: @superboyiii In all of them please

Comment (superboyiii) [reply neo-cli/CLI/MainService.Contracts.cs]: > @superboyiii In all of them please

Already done. Maybe you need refresh.

**Review by shargon — APPROVED** [review]

**Pull request merged**